grakn-core/src/main/java/ai/grakn/util/Schema.java

```diff
@@ -128,13 +125,12 @@ public static boolean isMetaLabel(Label label) {
         RELATIONSHIP_TYPE(RelationshipType.class),
         ATTRIBUTE_TYPE(AttributeType.class),
         ENTITY_TYPE(EntityType.class),
-        RULE_TYPE(RuleType.class),
+        RULE_TYPE(Rule.class),
```

Comment:
`RULE` I guess? But this is internal so less of an issue.

Reply:
🍌